goodies/bash/_zapier

```diff
@@ -31,7 +31,7 @@ _zapier() {
                 COMPREPLY=( $( compgen -W "--dest --entry" -- $cur ) )
                ;;
             test)
-                COMPREPLY=( $( compgen -W "--debug --timeout" -- $cur ) )
+                COMPREPLY=( $( compgen -W "--debug --timeout --skip-validate" -- $cur ) )
```

Comment:
I'm not sure I'd expect for test to skip validation. I might have missed the discussion, though.